### ISSUE TITLE
ERC20 Problem when used as an offer (Problem with latest seaport-js version)

### ISSUE THREAD

**Issue by MiguelGonzales-WIV** [ProjectOpenSea/seaport-js#221]
```
const { executeAllActions } = await seaport.createOrder(
{
	offer: [
		// buyer's offer
		{
			token: '0xc1d6ef502ac5410b3f3706beb6a0808131337fb6',
			amount: offerPrice,
		}
	],
	consideration: [
		// buyer's ask
		{
			itemType: ItemType.ERC721,
			token: smartContractAddress,
			identifier: tokenID,
			recipient: address, // recipient for token is the buyer
		},
	],
},
address
)
```
I recently updated my `@opensea/seaport-js` to the latest version and now my code doesn't work.

Using this block of code to try and create an offer for some NFT. Thing is, when I put my token in the offer, a Metamask error pops up, indicating "Invalid token value; should be exactly 64 hex digits long (u256)".

If I change the offer to properties that represent an ERC721 token, it works totally fine.
Also, removing `token` property from the object in offer array, it pushes through.

I need to be able to indicate which ERC20 token address I want as an offer, but it isn't working.

Hope to get some help!

**Comment by HeptaneL** [comment]
You can manually approve the ERC20 token to the seaport contract, and then execute the createOrder function.

**Comment by ryanio** [comment]
This should be resolved now with [v1.2.1](https://github.com/ProjectOpenSea/seaport-js/releases/tag/v1.2.1)
